sdks/python/apache_beam/transforms/core.py

```diff
@@ -882,6 +882,9 @@ def make_fn(self, fn):
   def default_label(self):
     return '%s(%s)' % (self.__class__.__name__, self._fn_label)
 
+  def process_argspec_fn(self):
+    return self.fn._fn  # pylint: disable=protected-access
```

Comment:
This should probably be self.fn.pocess_argspec_fn (or something similar) as this only handles the case of CallableWrapperCombineFn.